spec/user_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  it 'is not valid without a name' do
+    user = User.new(name: nil)
+    expect(user).to_not be_valid
+  end
+
+  it 'is not valid with a negative posts_counter' do
+    user = User.new(posts_counter: -1)
+    expect(user).to_not be_valid
+  end
```

Comment:
- [ ] Any custom method in your models should have corresponding tests. For example, the `three_most_recent_posts` method in your User model should've tested

Reply:
![image](https://github.com/lerfast/blog-app/assets/111823606/237308d0-4369-42ee-a1ce-036820a06521)
did it!!